test/LtiLibrary.AspNetCore.Tests/Outcomes/v1/OutcomesControllerShould.cs

```diff
@@ -70,6 +70,17 @@ public OutcomesControllerShould()
             Assert.True(replaceResult.StatusCode == HttpStatusCode.Unauthorized, $"{replaceResult.StatusCode} == {HttpStatusCode.Unauthorized}");
         }
 
+        [Theory]
+        [InlineData(null)]
+        [InlineData(-.01)]
+        [InlineData(1.01)]
+        public async void NotReplaceResult_IfScoreIsInvalid(double? score)
+        {
+            // First verify that plumbing works if secret is correct
```

Comment:
This comment does not describe what is about to happen. Copy and paste artifact?

Reply:
Whoops, yep.  Copied from the adjacent test.